### PR TITLE
SIG-1562 Filter styling modifications

### PR DESCRIPTION
**Pull request by janjaap**
This PR:
- adds two polyfills; `FormData` and `NodeList.forEach`
- has polyfill loading handled by Webpack instead of bluntly importing it in `app.js`
- upgrades the `@datapunt/asc-` packages to their latest versions
- uses SVG icons from `@datapunt/asc-assets` as intended (see https://amsterdam.github.io/amsterdam-styled-components for reference)
- makes sure that button labels are shown in the correct font
- renames and reorders menu items in the `SiteHeader` component
- corrects the date input placeholder text for the `FilterForm` component

### CODE REVIEW

File: src/components/Modal/index.js [hunk 1]
```diff
@@ -67,15 +61,16 @@ const Modal = ({ children, title, isOpen, onClose, ...rest }) => (
           <Heading as="h2">{title}</Heading>
         </Column>
 
-        <CloseButton
+        <Button
           data-testid="closeBtn"
           square
           type="button"
-          as="button"
```
Comment (jasperswart) [inline src/components/Modal/index.js]: add $as

File: src/components/PageHeader/index.js [hunk 2]
```diff
@@ -45,20 +52,24 @@ const PageHeader = ({ className, children, title }) => {
     };
   });
 
+  const StyledHeading = styled(Heading)`
+    font-size: 20px;
+  `;
```
Comment (jasperswart) [inline src/components/PageHeader/index.js]: you should also add line-height I think

File: src/components/SiteHeader/index.js [hunk 3]
```diff
@@ -5,36 +5,28 @@ import styled from 'styled-components';
 import Media from 'react-media';
 
 import CONFIGURATION from 'shared/services/configuration/configuration';
-import LoginIcon from '@datapunt/asc-assets/lib/Icons/Login.svg';
-import LogoutIcon from '@datapunt/asc-assets/lib/Icons/Logout.svg';
+import { Login as LoginIcon, Logout as LogoutIcon } from '@datapunt/asc-assets';
 import {
   Header as HeaderComponent,
+  MenuButton,
```
Comment (jasperswart) [inline src/components/SiteHeader/index.js]: I also put the MenuButton in the asc 0.12.7 header in my branch

File: src/components/Modal/index.js (hunk 1 above)
Comment (janjaap) [reply src/components/Modal/index.js]: Isn't necessary; component already renders a `<button>` element. `$as` Would only be required if I would want to render the `Button` component as a `<span>`, for instance.

File: src/components/PageHeader/index.js (hunk 2 above)
Comment (janjaap) [reply src/components/PageHeader/index.js]: `line-height` Has already been provided by the `asc-ui` package. Don't need to set it again.

**Comment by janjaap** [comment]
@jasperswart Please review again; comments addressed.

**Pull request merged**